config/setup/errors.go

```diff
@@ -21,12 +23,17 @@ func Errors(c *Controller) (middleware.Middleware, error) {
 	// Open the log file for writing when the server starts
 	c.Startup = append(c.Startup, func() error {
 		var err error
-		var file *os.File
+		var file io.Writer
 
 		if handler.LogFile == "stdout" {
 			file = os.Stdout
 		} else if handler.LogFile == "stderr" {
 			file = os.Stderr
+		} else if handler.LogFile == "syslog" {
+			file, err = gsyslog.NewLogger(gsyslog.LOG_ERR, "SYSLOG", "caddy")
```

Comment:
`SYSLOG` is probably not the correct *facility* for caddy's logging output, being the *facility* employed for messages generated by *syslogd*(8) itself. Cf., [*syslog*(3)](http://man7.org/linux/man-pages/man3/syslog.3.html).

Caddy should probably use one of the `LOCAL^[0-9]` facilities.